data/coalition/resistance intro.txt

```diff
@@ -1753,6 +1909,8 @@ mission "Resistance: Join"
 	on complete
 		set "language: Coalition"
 		event "first resistance unlock"
+		log "People" "Elirom" "While extremely disappointed with the ending of the interview Oobat conducted with the Quarg, Elirom seemed intrigued by the change in the Quarg's skin colour. He seemed to think it was linked to some emotion."
+		log "Returned with Oobat to the other Lunarium leaders, with the recorded interview. Tummug will work on editing it into a more effective propaganda piece, and was given a Coalition translator by Chiree having become an official member of the Lunarium Redoubt."
```

Comment:
I think it makes more sense for the translator bit to be it's own separate period sentence thingy.

```suggestion
		log "Returned with Oobat to the other Lunarium leaders, with the recorded interview, which Tummug will edit into a more effective propaganda piece. Was also given a Coalition translator by Chiree, having become an official member of the Lunarium Redoubt."
```